build.assets/tooling/cmd/difftest/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/gravitational/trace"
+	"gopkg.in/alecthomas/kingpin.v2"
```

Comment:
Can we use our fork for consistency? github.com/gravitational/kingpin

Reply:
Fixed. Did not know it exists.